### PR TITLE
:seedling: Including all packages in coverage

### PR DESCRIPTION
**Pull request by laozc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -470,7 +470,7 @@ test-junit: $(SETUP_ENVTEST) $(GOTESTSUM) $(GOVC) ## Run unit tests
 
 .PHONY: test-cover
 test-cover: ## Run unit tests and generate a coverage report
-	$(MAKE) test TEST_ARGS="$(TEST_ARGS) -coverprofile=coverage.out"
+	$(MAKE) test TEST_ARGS="$(TEST_ARGS) -coverpkg=./apis/...,./controllers/...,./pkg/...,./internal/... -coverprofile=coverage.out"
```
Comment (killianmuldoon) [inline Makefile:473]: Would all packages be covered by `./...` instead of naming each package individually.

Comment (sbueringer) [reply Makefile:473]: I think it should work

**Comment by sbueringer** [comment]
Interesting that this increases our coverage :D I would have expected the reverse

Details: https://app.codecov.io/gh/kubernetes-sigs/cluster-api-provider-vsphere/pull/2350/indirect-changes

File: Makefile (hunk 1 above)
Comment (killianmuldoon) [reply Makefile:473]: I believe so. IMO we should include the entire codebase for our coverage baseline.

Comment (sbueringer) [reply Makefile:473]: Yup will include that ones as well. I think it's fine to include all code. We just shouldn't use the one number to judge anything by. But we would always have the problem. There are more packages where we could discuss if they require test coverage or not. I think in the end we have to look package by package where we should add more tests and we can just skip packaging there.

**Comment by laozc** [comment]
It looks like we're doing quite good with just one line change.

**Comment by sbueringer** [comment]
Thank you!

Yup looks a lot better than expected already

/lgtm
/approve

**Pull request merged**